.buildkite/scripts/steps/trigger_pipeline.ts

```diff
@@ -11,13 +11,17 @@ import { BuildkiteClient } from '#pipeline-utils';
 const pipelineSlug = process.argv[2];
 const branch = process.argv[3] || 'main';
 const commit = process.argv[4] || 'HEAD';
+const kibanaBuildId = process.argv[5] || '';
 
 (async () => {
   try {
     const client = new BuildkiteClient();
     const build = await client.triggerBuild(pipelineSlug, {
       commit,
       branch,
+      env: {
+        KIBANA_BUILD_ID: kibanaBuildId,
+      },
```

Comment:
Do you think its safer to not pass this object if `kibanaBuildId` is an empty string? I see some other scripts looking for `KIBANA_BUILD_ID`, so just wanting to be safe.

Reply:
I pushed https://github.com/elastic/kibana/pull/159162/commits/73214c03a375c201172a1d426179dc33eeebaed1